packages/core/usage-data/core-usage-data-server-internal/src/core_usage_stats_client.test.ts

```diff
@@ -58,6 +61,67 @@ describe('CoreUsageStatsClient', () => {
   }; // as long as these header fields are truthy, this will be treated like a first-party request
   const incrementOptions = { refresh: false };
 
+  beforeAll(() => {
+    jest.useFakeTimers();
+  });
+
+  afterEach(() => {
+    stop$.next();
+  });
+
+  describe('Request-batching', () => {
+    [
+      { triggerName: 'timer-based', triggerFn: async () => await jest.runOnlyPendingTimersAsync() },
+      {
+        triggerName: 'forced-flush',
+        triggerFn: (usageStatsClient: CoreUsageStatsClient) => {
+          // eslint-disable-next-line dot-notation
+          usageStatsClient['flush$'].next();
+        },
+      },
+    ].map(({ triggerName, triggerFn }) => {
```

Comment:
FWIW jest has built in support for table-style tests: https://jestjs.io/docs/api#testeachtablename-fn-timeout

Reply:
Switched to Jest's table-style tests in [`fcde9fc` (#180405)](https://github.com/elastic/kibana/pull/180405/commits/fcde9fcfa3d1509320e5870137c414233443c174)